### PR TITLE
[Mobile-7496] Update dexguard support version (hold for v6.6.0)

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - Update dexguard support version for release 6.6.0

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review

 **DO NOT MERGE UNTIL 6.6.0 IS RELEASED**

**Comment by ywang-nr** [comment]
Approved by team lead, waiting for GA to merge

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-660.mdx
Comment (cthomas-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-660.mdx]: I added it.

**Review by cthomas-newrelic — APPROVED** [review]

**Review by cthomas-newrelic — DISMISSED** [review]

**Comment by cthomas-newrelic** [comment]
Hi @barbnewrelic ! Hope you had a great recharge at the offsite. This PR is ready to get whenever you can get to it. Thank you.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-660.mdx
Comment (barbnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-660.mdx:3]: This PR looks fine! My only question is, the releaseDate says June 9. Do you want us to wait until the 9th to merge this PR, or would you like to change the date and release today? Thanks for confirming either way!

**Review by barbnewrelic — DISMISSED** [review]

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-660.mdx
Comment (cthomas-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-660.mdx:3]: Oh sorry, we can make it June 2. That was probably just the projected release date.

**Comment by barbnewrelic** [comment]
Congrats on your latest release!

**Comment by cthomas-newrelic** [comment]
Thanks for all your help!

**Pull request merged**
